webapp/src/pages/PageNotFound/PageNotFound.tsx

```diff
@@ -1,11 +1,15 @@
 import React from "react";
 import s from "./PageNotFound.module.scss";
 import { Link } from "react-router-dom";
+import Logo from "../../assets/images/page_not_found.svg"
 
 const PageNotFound = () => {
   return (
-    <div>
-      PageNotFound <Link to="/">Return to Home Page</Link>;
+    <div className={s.root}>
+      <img src={Logo} alt="logo" />
+      <button>
+      <Link to="/">Return to the main page</Link>
```

Comment:
Link creates an <a></a> tag, which is not welcomed inside button tag cause accessibility reasons, please use only Link or only button, or <Link><button>btn</button><Link>